helpers/config.py

```diff
@@ -780,6 +793,50 @@ def __questions_aws(self):
             self.__dict['aws_secret_key'] = ''
             self.__dict['aws_bucket_name'] = ''
 
+    def __questions_aws_validate_credentials(self):
+        """
+        Prompting user whether they would like to validate their entered AWS
+        credentials or continue without validation.
+        """
+        aws_credential_attempts = 0
+
+        if self.__dict['use_aws']:
+            self.__dict['aws_validate_credentials'] = CLI.yes_no_question(
+                'Would you like to validate your AWS credentials?',
+                default=self.__dict['aws_validate_credentials'],
+            )
+
+        if self.__dict['use_aws'] and self.__dict['aws_validate_credentials']:
+            while not self.__dict['aws_credentials_valid'] and (
```

Comment:
I think this while-statement is incorrect. If user has previously enter correct credentials, the block will be skipped all the time (i.e. `self.__dict['aws_credentials_valid']` will be always `True`)

Reply:
That's a great point! 🤦